### ISSUE TITLE
Expose PeaksViewer Fully

### ISSUE THREAD

**Issue by OwenArnold** [mantidproject/mantid#8357]
This issue was originally [TRAC 7512](http://trac.mantidproject.org/mantid/ticket/7512)

This ticket is blocked by : 
- [X] [TRAC7013](http://trac.mantidproject.org/mantid/ticket/7013)
- [ ] [TRAC7512](http://trac.mantidproject.org/mantid/ticket/7512)
- [X] [TRAC8381](http://trac.mantidproject.org/mantid/ticket/8381)

This ticket is blocks :  [TRAC7512](http://trac.mantidproject.org/mantid/ticket/7512)

We didn't have enough time in iteration 2.6 to do anything other than a quick implementation. The full and proper implementation would be to expose the PeaksViewer via the SliceViewerWindow as an object in it's own right.

A complication here is that the SliceViewer (not the PeaksViewer widget itself), creates the necessary presenters (MVP) to control the GUI. This would suggest that using forwarding methods from the PeaksViewer widget (exposed via SIPs) to the PeaksPresenters would be the correct thing to do.

Once we can instantiate a PeaksViewer object in python, select methods should be provided for exposure on that type directly.

3 Days

**Comment by Mantid-Matthew** [comment]
As slice-viewer is being rewritten this is no longer relevant.
